modules/claims_api/app/swagger/claims_api/description/v2.md

```diff
@@ -28,25 +28,25 @@ End-to-end claims tracking provides the status of claims as they move through th
 
-Claims are first submitted by this API and then established in Veterans Benefits Management System (VBMS). A 200 response means that the claim was successfully submitted by the API. It does not mean VA has received the claim. Use the appropriate GET endpoint and the ID returned with your submission response to confirm the status of the submission. Statuses are:
+After you submit a disability compensation claim with the ‘POST /veterans/{veteranId}/526’ synchronous endpoint, it is then established in Veterans Benefits Management System (VBMS). A ‘202’ response means that the claim was successfully submitted by the API. However, it does not mean VA has received the required 526EZ PDF. 
 
--   Pending - the claim is successfully submitted for processing
--   Errored -  the submission encountered upstream errors
--   Canceled -  the claim was identified as a duplicate or another issue caused the claim to be canceled. For duplicate claims,  the tracking of the claim's progress happens under a different Claim ID . 
+To confirm the status of your submission, use the ‘GET /veterans/{veteranId}/claims/{id}’ endpoint and the ID returned with your submission response. Statuses are: 
 
-Other statuses this API returns align with the [VA.gov](http://va.gov/) [claim status descriptions](https://www.va.gov/resources/what-your-claim-status-means/), which are:
-
--   Claim received
--   Initial review
--   Evidence gathering, review, and decision
--   Preparation for notification
--   Complete
+-  **Pending**: The claim is successfully submitted for processing
+-   **Errored**: The submission encountered upstream errors
+-   **Canceled**: The claim was identified as a duplicate, or another issue caused the claim to be canceled. 
+  -   For duplicate claims, the claim's progress is tracked under a different Claim ID than the one returned in your submission response. 
+-   **Claim received**: The claim was received, but hasn't been assigned to a reviewer yet.
+-   **Initial review**: The claim has been assigned to a reviewer, who will determine if more information is needed.
+-   **Evidence gathering, review, and decision**: VA is gathering evidence to make a decision from health care providers, government agencies, and other sources.
+-   **Preparation for notification**: VA has made a decision on the claim, and is getting a decision letter ready to mail.
+-   **Complete**: VA has sent a decision letter by U.S. mail.
 
 ### Finding a Veteran's unique VA ID
 
-This API uses a unique Veteran identifier to identify the subject of each API request. This Veteran identifier can be retrieved by passing the Veteran’s first name, last name, DOB, and SSN to the ‘/veteran-id’ endpoint. This identifier should then be used as the Veteran ID parameter in request URLs.
+This API uses Inegration Control Number (ICN) as a unique Veteran identifier to identify the subject of each API request. This identifier should be used as the ‘{veteranId}’ parameter in request URLs.
```

Comment:
should it backticks around veteranId for a code block? or * for italic? this just wraps it in '{veteranId}' vs `{veteranId}` or *{veteranId}*